README.md

```diff
@@ -138,25 +142,28 @@ Or by setting the environment variable `BSK_SUBDOMAIN_PROMETHEUS_PORT`
 Queue a registration:
 
 ```bash
-$ curl -X POST -H 'Authorization: bearer API-KEY-IF-USED' -H 'Content-Type: application/json' --data '{"zonefile": "$ORIGIN spqr\n$TTL 3600\n_https._tcp URI 10 1 \"https://gaia.blockstack.org/hub/1HgW81v6MxGD76UwNbHXBi6Zre2fK8TwNi/profile.json\"\n", "name": "spqr", "owner_address": "1HgW81v6MxGD76UwNbHXBi6Zre2fK8TwNi"}' http://localhost:3000/register/
+curl -X POST -H 'Authorization: bearer API-KEY-IF-USED' -H 'Content-Type: application/json' --data '{"zonefile": "$ORIGIN spqr\n$TTL 3600\n_https._tcp URI 10 1
+```
+```
+\"https://gaia.blockstack.org/hub/1HgW81v6MxGD76UwNbHXBi6Zre2fK8TwNi/profile.json\"\n", "name": "spqr", "owner_address": "1HgW81v6MxGD76UwNbHXBi6Zre2fK8TwNi"}' http://localhost:3000/register/
 ```
 
 Force a batch:
 
 ```bash
-$ curl http://localhost:3000/issue_batch -X POST -H 'Authorization: bearer PASSWORDHERE'
+curl http://localhost:3000/issue_batch -X POST -H 'Authorization: bearer PASSWORDHERE'
 ```
 
 Force zonefile check:
 
 ```bash
-$ curl http://localhost:3000/check_zonefile -X POST -H 'Authorization: bearer PASSWORDHERE'
+curl http://localhost:3000/check_zonefile -X POST -H 'Authorization: bearer PASSWORDHERE'
 ```
 
 Check subdomain status:
 
 ```bash
-$ curl http://localhost:3000/status/spqr | jq .
+curl http://localhost:3000/status/spqr | jq .
```

Comment:
For this one, we should leave it as is since it shows the command *and* the output.